metrics-proxy/src/main/java/ai/vespa/metricsproxy/node/HostLifeGatherer.java

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package ai.vespa.metricsproxy.node;
+
+import ai.vespa.metricsproxy.metric.Metric;
+import ai.vespa.metricsproxy.metric.model.ConsumerId;
+import ai.vespa.metricsproxy.metric.model.MetricId;
+import ai.vespa.metricsproxy.metric.model.MetricsPacket;
+import ai.vespa.metricsproxy.metric.model.ServiceId;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.time.Instant;
+import java.util.List;
+import java.util.Set;
+import java.util.logging.Logger;
+
+
+/**
+ * @author olaa
+ */
+public class HostLifeGatherer {
+
+    private static final Path UPTIME_PATH = Path.of("/proc/uptime");
```

Comment:
This wont work inside docker containers: returns the value for the host.

Reply:
This is probably also Linux specific. I suggest to use `SystemPollerProvider.runningOnLinux` to switch this functionality on/off to avoid log warnings on other OS's.

Reply:
... so this should probably move to host-admin, and read the "created time" of the Docker container.

Reply:
Or just do what we do for container MotD: last modified time of `/proc`

Reply:
That seems to work outside Docker containers too.